readme.md

```diff
@@ -35,14 +52,18 @@ Type: `string`
 
 String with CSS.
 
+
 ## options
 
-### all
+### preserve
 
-Type: `boolean`  
-Default: `false`
+Type: `boolean`, `RegExp`, or `function` 
+Default: `true`
 
-Whether *important* CSS comments *(those starting with `/*!`)* should be stripped.
+- `preserve: true` &mdash; (default) preserve comments that use the `/*! */` syntax;
```

Comment:
I guess it might be more grammatically correct, but I prefer the look of the normal dash.

https://en.wikipedia.org/wiki/Dash#Em_dash

You also shouldn't use spaces around an em-dash.